templates/tags.tt

```diff
@@ -0,0 +1,20 @@
+[% WRAPPER wrapper.tt title = 'Tags' %]
```

Comment:
The tag-specific pages look good, but I'm not sure about the top-level tag index. Will post thoughts on the overall review.